client-misk-dynamodb/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=client-misk-dynamodb
+POM_NAME=client-misk-dynamodb
+POM_DESCRIPTION=Misk DynamoDb client library for backfila
```

Comment:
Well it relies on the misk-client stuff. But otherwise yeah. I'm thinking it might make sense to eventually extract the misk part out. But for now it is misk specific.